lib/travis/build/appliances/wait_for_network.rb

```diff
@@ -5,6 +5,7 @@ module Build
     module Appliances
       class WaitForNetwork < Base
         def apply
+          return if Travis::Build.config.network_available
```

Comment:
See comment in config.rb
I propose to change it to:
`return unless Travis::Build.config. wait_for_network_check`